src/controllers/jiraFieldMap.gs

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Unit tested and manually tested (except for the error response and Browser pop up)